### ISSUE TITLE
SketchUp favicon URL has changed causing error and standard browser tab icon

### ISSUE THREAD

**Issue by DanRathbun** [SketchUp/sketchup-yard-template#21]
SketchUp favicon URL has changed causing error and standard "document" browser tab icon.

> Per [Caroline in the public forum](https://forums.sketchup.com/t/favicon-seems-to-be-missing/57625/9)
>
> This is the link: https://www.sketchup.com/sites/all/themes/sketch_www/favicon.ico

... it is currently ...

`https://help.sketchup.com/sites/all/themes/sketch_help/favicon.ico`

... which gives a `404` Not Found error.

----
_Reference public forum thread:_
https://forums.sketchup.com/t/favicon-seems-to-be-missing/57625/

**Comment by DanRathbun** [comment]
This is either still busted or busted again !

**Comment by thomthom** [comment]
Yup, I noticed that. Got a fix in the pipeline which pulls the favicon into the template itself instead of relying on external resources.

**Comment by thomthom** [comment]
Fix was done in this commit: https://github.com/SketchUp/sketchup-yard-template/commit/fa19368c40e2525c38d67c1a4bc800e408022183

But we haven't pushed any update yet. Working on some other fixes as well.

**Comment by DanRathbun** [comment]
**I think has been fixed.** We now see the new blue SketchUp application icon in browser tabs.

**Comment by DanRathbun** [comment]
Actually, ... the icon has a permanent white background even when a dark mode is used.

Please **replace** the icon here: `"https://ruby.sketchup.com/favicon.ico`"
with this icon: `"https://developer.sketchup.com/sites/developer.sketchup.com/files/Favicon.ico"`
or this: `"https://www.sketchup.com/themes/sketchup_www_terra/favicon.ico"`
or this: `"https://aws1.discourse-cdn.com/sketchup/optimized/3X/c/7/c746725cdbfbabf601da75a1af32622133cfacdd_2_32x32.png"`

... ie, they are all the same 32x32 icon. Although the last is a PNG.
